src/app/components/DynamicContentStyles.tsx

```diff
@@ -22,27 +24,35 @@ const DynamicContentStyles = React.forwardRef<HTMLElement, DynamicContentStylesP
 ) => {
   const queryParams = useSelector(query);
   const [styles, setStyles] = React.useState('');
+  const currentBook = useSelector(book);
 
   React.useEffect(() => {
     if (disable) {
       setStyles('');
       return;
     }
 
-    const cssfileUrl = queryParams['content-style'];
+    let cssfileUrl = queryParams['content-style'];
+    if (!cssfileUrl && currentBook) {
+      const bookConfig = BOOKS[currentBook.id];
+      if (bookConfig && bookConfig.dynamicStyles) {
+        cssfileUrl = currentBook.style_href;
```

Comment:
you gotta run this through that content reference resolver helper thing don't you?

Reply:
like this, to rebase the relative url based on the content url? https://github.com/openstax/rex-web/blob/master/src/app/content/utils/contentManipulation.ts#L30

Reply:
I got into several rabbit holes trying to get the book archive url from the book object, but in the end building it from REACT_APP_ARCHIVE_URL seems to have worked...